gui/slick/views/layouts/main.mako

```diff
@@ -117,6 +117,12 @@
                                 <li><a href="${srRoot}/home/"><i class="menu-icon-home"></i>&nbsp;Show List</a></li>
                                 <li><a href="${srRoot}/home/addShows/"><i class="menu-icon-addshow"></i>&nbsp;Add Shows</a></li>
                                 <li><a href="${srRoot}/home/postprocess/"><i class="menu-icon-postprocess"></i>&nbsp;Manual Post-Processing</a></li>
+                                % if sickbeard.SHOWS_RECENT:
```

Comment:
It isnt needed in config.ini. If the user hasnt visited any shows yet there is nothing shown in the nav (its hidden).  If SR just started the user hasnt displayed any shows.